js/lib/cli.js

```diff
@@ -326,6 +349,12 @@ function processInputSync(filepath) {
 
 function makePretty(code, config, outfile, callback) {
     try {
+        if (config.editorConfig) {
```

Comment:
@bitwiseman https://github.com/beautify-web/js-beautify/blob/master/js/lib/cli.js#L182 starting here, the configuration is set up prior to inspecting into each individual file. I cannot hook up the editorconfig configs here since I have no idea about the specific one file that will be beautified. But I also cannot hook up right before makePretty since the default settings already kick in. Unless we rewire the logic of the config step I doubt if there is a simple way to allow individual settings to override editorconfig easily